<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExampleWidget</class>
 <widget class="QWidget" name="ExampleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>596</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="border" native="true">
     <property name="styleSheet">
      <string notr="true">#border {
	border: 1px solid gray;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="title_bar" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="title" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="title_icon">
              <property name="styleSheet">
               <string notr="true">QPushButton {
	color: black;
	border: hidden;
	background-color: transparent;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="title_title">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>440</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="winbutton" native="true">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: transparent;
	border: hidden;
	width: 5em;
	height: 100%;
	color: black;
}

QPushButton:hover {
	background-color: rgba(80,80,80,0.1);
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QPushButton" name="winbutton_min">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="winbutton_toggle">
              <property name="text">
               <string>O</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="winbutton_close">
              <property name="styleSheet">
               <string notr="true">QPushButton:hover {
	background-color: rgba(255,0,0,0.6);
}</string>
              </property>
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="body" native="true">
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>110</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>test1</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>190</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>test2</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>110</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>test3</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>240</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>test5</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>150</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>test4</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>290</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>test8</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_7">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>330</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>test7</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="geometry">
          <rect>
           <x>580</x>
           <y>270</y>
           <width>80</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>test6</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
